CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# How to contribute to this project ?
+
+1. First you must choose a issue on [the project github](https://github.com/mdl29). If you want to do something but no issue were create yet you must create one (try to tag it properly).
+2. Fork the project.
+3. Create a new branch whith the following name format on your fork : `<issueNumber>_<descriptiveNameOfTheIssue>`.
+
+> Example : if you wanted to solve this issue (image below) you could have named your branch `138_Kickstarter_Yanis`.
+> ![example issue](./doc/example_issue.png)
+
```

Comment:
Add recommended format for commit messages that should reference the issue, for instance : 
Recommended format for the commit(s) message(s):
- resolves #ISSUEID Component: title for features
- fixes #ISSUEID Component: title for fixes

Reply:
Component are : backend / frontend / tools / doc

Reply:
Branch name convention: **ISSUEID_issue-title-message**.